pom.xml

```diff
@@ -50,15 +51,15 @@
 
 	<repositories>
 		<repository>
-			<id>Eclipse 4.13 (2019-09)</id>
+			<id>Eclipse 4.16 (2020-06)</id>
 			<layout>p2</layout>
 			<!--<url>http://download.eclipse.org/releases/2019-09</url> -->
 			<url>http://ftp-stud.hs-esslingen.de/pub/Mirrors/eclipse/releases/2019-09/</url>
```

Comment:
We should update the base repository to 2020-06 as well, although everything should be available in the updates repository anyway.

Reply:
Right. That slipped through. Fixed.